semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SymbolInformationOps.scala

```diff
@@ -226,13 +226,18 @@ trait SymbolInformationOps { self: SemanticdbOps =>
       }
     }
 
+    def overriddenSymbols: List[String] = {
+      gsym.allOverriddenSymbols.map(_.toSemantic)
```

Comment:
Should we make this behavior configurable? It should be enabled by default, but it might be helpful to have an option to disable it in case there are bugs we're not aware of. Not a blocking issue, just an idea.

Reply:
Good idea, added. Let me know what you think @olafurpg

Reply:
Looks great!